DataManagementSystem/Utilities/CatalogUtilities.py

```diff
@@ -0,0 +1,138 @@
+########################################################################
+# $HeadURL$
+########################################################################
+
+""" DIRAC FileCatalog utilities
```

Comment:
Why is this file in DataManagementSystem? Shouldn't it rather be in /Resources/Utilities?